opencti-platform/opencti-front/src/private/components/analyses/external_references/AddExternalReferences.jsx

```diff
@@ -62,15 +65,32 @@ const AddExternalReferences = ({
         <Add fontSize="small" />
       </IconButton>
       <Drawer
-        title="Add external references"
+        title={t_i18n('Add external references')}
         open={open}
         onClose={handleClose}
         header={(
-          <div className={classes.search}>
+          <div style={{
+            marginLeft: 'auto',
+            marginRight: '20px',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-end',
+          }}
+          >
             <SearchInput
               variant="inDrawer"
               onSubmit={handleSearch}
             />
+            {FABReplaced
+              && <Button sx={{ marginTop: '5px' }}
+                onClick={() => setDialogOpen(true)}
+                color='primary'
+                size='small'
+                variant='contained'
+                 >
+                {t_i18n('Create')} {t_i18n('entity_External-Reference')} <Add />
```

Comment:
Still the icon here

Reply:
![image](https://github.com/OpenCTI-Platform/opencti/assets/33682259/ec15a404-2c20-41fb-ba77-70ec5da2c222)
And small alignment issue with the search input and button